WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/TitleAndTextFieldRow.swift

```diff
@@ -58,6 +64,33 @@ private extension TitleAndTextFieldRow {
     enum Constants {
         static let height: CGFloat = 44
         static let padding: CGFloat = 16
+        static let spacing: CGFloat = 20
+    }
+}
+
+/// PreferenceKey to store max title width among the fields.
+///
+struct MaxWidthPreferenceKey: PreferenceKey {
+    static var defaultValue: CGFloat? = nil
+
+    static func reduce(value: inout CGFloat?, nextValue: () -> CGFloat?) {
+        if let nv = nextValue(), nv > value ?? .zero {
+            value = nv
+        }
+    }
+}
+
+private struct MaxWidthModifier: ViewModifier {
+    private var sizeView: some View {
+        GeometryReader { geometry in
+            Color.clear
+                .preference(key: MaxWidthPreferenceKey.self,
+                            value: geometry.size.width)
```

Comment:
Not a fan of helper views, but wasn't able to find a better solution.
There are "Anchor preferences" that implicitly read view geometry, but they are more complicated and doesn't provide single "width" or "layout guide" value. They fit best to convert rect/point from one view context to another, but we don't need it here.